examples/pachyderm/sklearn_pachyderm.ipynb

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
<p>Best would be to point them to their docs and tell them to sort out downloadng the client themselves to avoid having to maintain it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1738/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Just pointing to their docs seems good indeed!